python/src/trezorlib/cli/cardano.py

```diff
@@ -58,11 +59,12 @@ def sign_tx(client, file, network):
 @cli.command()
 @click.option("-n", "--address", required=True, help=PATH_HELP)
 @click.option("-d", "--show-display", is_flag=True)
+@click.option("-N", "--network", type=int, default=1)
 @with_client
-def get_address(client, address, show_display):
+def get_address(client, address, show_display, network):
     """Get Cardano address."""
     address_n = tools.parse_path(address)
-    return cardano.get_address(client, address_n, show_display)
+    return cardano.get_address(client, address_n, show_display, network)
```

Comment:
Do we call
```
def get_address(
    client, address_n: List[int], protocol_magic: int, show_display=False
) -> messages.CardanoAddress:
```
?

If yes, I would not call the command line arg `--network` when it is passed as `protocol_magic`, it just adds to the confusion IOHK created.

Reply:
Very true.